### DISCUSSION TITLE
Speed ​​feedback

### DISCUSSION THREAD

**Discussion by Vali6053** [siku2/hass-dingz#45] [General]
MQTT is active for me and it sometimes takes 3-4 seconds until a response is received from the Dingz to the home assistant.
If I switch on a light in the Home Assistant that is connected to the Dingz and try to switch it off again after 3 seconds, then nothing responds for 5 to 8 seconds and I always have to wait. What could that be?

**Comment by siku2** [comment]
Relevant issue: https://github.com/siku2/hass-dingz/issues/34
